src/netius/extra/proxy_c.py

```
@@ -0,0 +1,283 @@
+                    self.auth[domain] = simple_auth
+            elif tag.startswith("proxy.error-url="):
+                error_url = tag[len("proxy.error-url=") :]
+                if error_url:
+                    self.error_urls[domain] = str(error_url)
+            elif tag == "proxy.redirect-ssl=true":
+                self.redirect[domain] = (domain, "https")
+
+    def _build_urls(self, instances):
+        urls = []
+        for instance in instances:
+            service = instance.get("Service", dict())
```

Pylint (4.0.4)

[refactor] Consider using '{}' instead of a call to 'dict'.

(R1735)